.github/workflows/static.yml

```
@@ -30,12 +30,4 @@ jobs:
       os: >-
         ['ubuntu-latest']
       php: >-
-        ['8.1', '8.2', '8.3', '8.4']
-  psalm80:
-    uses: yiisoft/actions/.github/workflows/psalm.yml@master
-    with:
-      psalm-config: psalm80.xml
-      os: >-
-        ['ubuntu-latest']
-      php: >-
-        ['8.0']
+        ['8.0', '8.1', '8.2', '8.3', '8.4']
```

```suggestion
        ['8.0', '8.1', '8.2', '8.3', '8.4', '8.5']
```
Does it support 8.5?

---

Psalm doesn't support 8.5, so it skipped here.